WcaOnRails/app/controllers/api/v0/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V0::PersonsController < Api::V0::ApiController
+  def me
```

Comment:
Don't we already have a `/me` route?!

Reply:
Yup, if we can recycle the same piece of code in two different places then go for it!

Reply:
That me action currently has `before_action :doorkeeper_authorize!, only: [:me]` though, so it might overcomplicate things if I merge the two?